openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -180,3 +206,57 @@ def _email_url_parser(url_name, extra_param=None):
     dashboard_url_path = reverse(url_name) + extra_param if extra_param else reverse(url_name)
     dashboard_link_parts = ("https", site_name, dashboard_url_path, '', '', '')
     return urlparse.urlunparse(dashboard_link_parts)
+
+
+def get_credit_providers_by_course(course_key):
```

Comment:
This function returns credit provider display names. Name the function accordingly (e.g., `get_credit_provider_display_names`). Fix the docstring, too; the function returns a list of credit provider display names.